internal/runners/packages/install.go

```diff
@@ -38,16 +38,21 @@ func NewInstall(prime primeable) *Install {
 // Run executes the install behavior.
 func (a *Install) Run(params InstallRunParams, nstype model.NamespaceType) error {
 	logging.Debug("ExecuteInstall")
+	var language string
+	var err error
 	if a.proj == nil {
-		return locale.NewInputError("err_no_project")
-	}
-
-	language, err := model.LanguageForCommit(a.proj.CommitUUID())
-	if err != nil {
-		return locale.WrapError(err, "err_fetch_languages")
+		language, err = model.LanguageForPackage(params.Package.Name())
+		if err != nil {
+			return locale.WrapError(err, "err_install_get_langauge", "Could not get language for package: {{.V0}}", params.Package.Name())
+		}
+	} else {
+		language, err = model.LanguageForCommit(a.proj.CommitUUID())
+		if err != nil {
+			return locale.WrapError(err, "err_fetch_languages")
+		}
```

Comment:
This is fundamentally flawed as we should not have any opinion on the language, that's what the solver is for.

Of course that does raise issues for certain State Tool use-cases, but we should be addressing those issues rather than working around them by spoofing the language.